README.md

```diff
@@ -1 +1,119 @@
 # egghead-instructor-center
+
+_An app for egghead instructors to do instructor things_
+
+[ ![Codeship Status for eggheadio/egghead-instructor-center](https://app.codeship.com/projects/3a4afe00-8808-0134-e6cc-2e5dfce30ebc/status?branch=master)](https://app.codeship.com/projects/183842)
+
+---
+
+# Getting started
+
+- Ensure you have Git, Node, and Yarn installed
+- Clone this repo and `cd` into it
+- `yarn` to install packages
+- `yarn test` to run tests
+- `yarn dev` to run app
+- Open [localhost:3000](http://localhost:3000) to view app
+
+---
+
+# Conventions
+
+## Package management
+
+Yarn is used for package management. Use the [yarn cli](https://yarnpkg.com/en/docs/usage) to add/remove/update packages which updates both the `package.json` and `yarn.lock` to ensure consistent package installs. 
+
+## Scripts
+
+Yarn is used for running scripts. Use `yarn {script}` to run them. The core of the scripts extend `react-scripts` (from Create React App) so that this project doesn't have to manage compiling, linting, or testing; it generally gets all [Create React App updates](https://github.com/facebookincubator/create-react-app/releases) for free by running `yarn upgrade react-scripts@latest`. Avoid ejecting the project for this reason if at all possible.
+
+## Components
+
+Each directory inside `src` is a **component**. A component is a directory organized _by feature_. They look like this:
+
+```
+SomeComponentName/
+    index.js (entry point)
+    components/ (optional sub-components)
```

Comment:
@tayiorbeii `egghead-ui` doesn't need to fit this unless you wanted it to; this is just a convention for this project. As long as this project can `yarn install egghead-ui` and `import {SomeComponent} from 'egghead-ui'`, the internals can be whatever / different from this project.